CONTRIBUTING.md

```diff
@@ -79,8 +69,10 @@ By making a contribution to this project, I certify that:
 * d. I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.
 
 ## Resources
-Aspects of our open source policy have been influenced from other well respected community projects.
-* [Project Governance @Node] : https://nodejs.org/en/about/governance/
-* [Contributions @Node] : https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
-* [Open Source @Github] : https://github.com/blog/2039-adopting-the-open-code-of-conduct
-* [Open Source exmaples @todogroup] : https://github.com/todogroup/policies
+
+Several open source projects have influenced our contribution policy:
+
```

Comment:
This has changed significantly enough that we probably could remove the "Resources section all together if you want" The previous one was aligned fairly closely with Node's policy.  This now is a simplified and original version unless you also closely aligned with some resource.

Reply:
Does not hurt to give them credit for our point of inspiration.